opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -109,6 +109,15 @@ public void init(FilterConfig filterConfig) throws ServletException {
         this.filterConfig = filterConfig;
         ServletContext servletContext = filterConfig.getServletContext();
 
+        // Check whether the servlet context provides a tracer
+        Tracer scTracer = (Tracer)servletContext.getAttribute(Tracer.class.getName());
```

Comment:
A safer approach is to get an Object and check if it's an instance of Tracer before casting.

```java
Object tracerObj = servletContext.getAttribute(Tracer.class.getName());
if (tracerObj instanceof Tracer) {
  // no need for the null check!
  tracer = (Tracer) tracerObj;
}
```